### DISCUSSION TITLE
Subdirectory under proxy

### DISCUSSION THREAD

**Discussion by Ricoldi** [chatwoot/chatwoot#10397] [Self hosted]
hello everybody,

I sorry, but I already searched everywhere possible before calling here.

I can't put it in a subdomain because I use a ddns. I need put chatoot in subdirectory "aaa.com/cw"

I'm running nginx as a reverse proxy, I've already tried several configurations with rewrite and without.

In chatwoot I have already tried several ways to leave the "RAILS_RELATIVE_URL_ROOT" and "FRONTEND_URL" parameters, even the "config.relative_url_root" parameter directly in the configuration file.

Either I get 404 because I don't have the full path, or the subdirectory is duplicated to times.

**Comment by sojan-official** [comment]
This setup isn’t supported. Chatwoot requires a fully qualified domain name or subdomain. Configuring it under a subdirectory will disrupt functionalities, including custom domains for portals and email forwarding. We don’t recommend this structure as it’s not aligned with typical use cases.

**Reply by Ricoldi** [reply]
It's a total shame, because I can't understand what the difference would be since, in my ignorant understanding, both cases are just redirections within the nginx reverse proxy, which I already use with everything today, including let's encrypt.

And as I said, I can't have a subdomain because I use dynamic DNS.

Im SAD :(
